bitcoin/electrumx/bin/init

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
Why is this file a binary instead of a script `init.sh`?

Reply:
It's a bit easier to run electrum this way and avoid juggling with paths. Also, there were some problems with interrupt signals while using shell script.